orderbook/openapi.yml

```diff
@@ -153,8 +153,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/FeeInformation"
-        400:
+        404:
```

Comment:
@fleupold - I wonder if the 400 here was intentional or was meant to be a 404 like the others.

Reply:
Yes, good catch this should be a 404